app/src/main/java/mozilla/lockbox/view/FxALoginFragment.kt

```diff
@@ -43,21 +45,37 @@ class FxALoginFragment : BackableFragment(), FxALoginView {
         if (!isDebug()) {
             skipFxA.visibility = View.GONE
         }
+
         return view
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        println("ELISETEST: onViewCreated hit")
         super.onViewCreated(view, savedInstanceState)
         view.toolbar.setNavigationIcon(R.drawable.ic_close)
     }
 
+    init {
+        println("ELISETEST: init FxaLoginFragment")
+    }
+
     override fun loadURL(url: String) {
+        println("ELISETEST: loadUrl hit")
```

Comment:
Nit: Happy to have logging, though you should probably use `log.debug()`. Best remove this sort of thing before committing :)

Reply:
`log.debug()`: noted! 👍 👍 👍